x/marker/spec/07_events.md

```diff
@@ -0,0 +1,173 @@
+| marker_cancelled       | administrator         | {admin account address}   |
+
+
+---
+## Destroy
+
+Fires when a marker is marked as destroyed and ready for removal.
+
+| Type                   | Attribute Key         | Attribute Value           |
+| ---------------------- | --------------------- | ------------------------- |
+| marker_destroyed       | denom                 | {denom string}            |
+| marker_destroyed       | administrator         | {admin account address}   |
+
+---
+## Mint
+
+Fires when coins are minted for a marker.
+
+| Type                   | Attribute Key         | Attribute Value           |
+| ---------------------- | --------------------- | ------------------------- |
+| marker_minted_coins    | denom                 | {denom string}            |
+| marker_minted_coins    | amount                | {supply amount}           |
+| marker_minted_coins    | administrator         | {admin account address}   |
+
+
+
+---
+## Burn
+
+Fires when coins are burned from a marker account.
+
+| Type                   | Attribute Key         | Attribute Value           |
+| ---------------------- | --------------------- | ------------------------- |
+| marker_burned_coins    | denom                 | {denom string}            |
+| marker_burned_coins    | amount                | {supply amount}           |
+| marker_burned_coins    | administrator         | {admin account address}   |
+
+
+---
+
+Fires when coin is removed from a marker account and transferred to another.
+## Withdraw
+
+| Type                   | Attribute Key         | Attribute Value           |
+| ---------------------- | --------------------- | ------------------------- |
+| marker_withdraw_coins  | denom                 | {denom string}            |
+| marker_withdraw_coins  | amount                | {supply amount}           |
+| marker_withdraw_coins  | administrator         | {admin account address}   |
+
+
+---
+## Transfer
+
+Fires when a facilitated transfer is performed of the marker's coin between accounts by an administrator
+
+| Type                   | Attribute Key         | Attribute Value           |
+| ---------------------- | --------------------- | ------------------------- |
+| marker_tranfer_coin    | denom                 | {denom string}            |
+| marker_tranfer_coin    | amount                | {supply amount}           |
+| marker_tranfer_coin    | administrator         | {admin account address}   |
+
+---
+## Withdraw Asset
+
+Fires when coins are withdrawn from the marker account and sent to an address by an administrator
+
+| Type                   | Attribute Key         | Attribute Value           |
+| ---------------------- | --------------------- | ------------------------- |
+| marker_asset_withdrawn | denom                 | {denom string}            |
+| marker_asset_withdrawn | amount                | {coins amount}            |
+| marker_asset_withdrawn | administrator         | {admin account address}   |
```

Comment:
would `to address` be helpful here?

Reply:
this entire event needs to be removed actually ... the withdraw coins above is valid, this one is not because the metadata module is handling the asset pools

Reply:
deleted.